webpack.config.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
     open: true,
     liveReload: true,
     hot: false,
+    host: "0.0.0.0",
```

Comment:
I can't see the started project. See here https://disk.yandex.ru/i/AAW3SSu53g9Lqg

Reply:
Try it now, it should work. I was working on deploying the website through render (https://iankamar-aroundtheus.onrender.com/), by disabling the host check in your webpack.config.js file by adding the following line to devServer object: disableHostCheck: true and changing the host property to property host: "0.0.0.0" to accept connections from all client's webpage, but I'm still receiving an error (Invalid Host header).